test/integration/comprehensive-transformer-test.spec.ts

```diff
@@ -75,19 +83,28 @@ describe('Comprehensive Transformer Test', () => {
         .registerTransformers([{ transformer: transformer.address, dependents: [TOKENS['WETH'].address] }]),
   });
 
+  transformerComprehensiveTest({
+    transformer: 'wstETHTransformer',
+    dependent: 'wstETH',
+    underlying: ['stETH'],
+    threshold: 2, // stETH can be lost on transfers, so we have a small 2-wei threshold
```

Comment:
Should we always send in the `minAmounts` from off-chain ~2 wei less? just so transactions don't revert 😬

Reply:
Yeah, we'll need to be careful when using this from off-chain, since it can be less than expected